pybossa/importers.py

```diff
@@ -86,7 +86,9 @@ def _get_csv_data_from_request(self, r):
             msg = gettext("Oops! That file doesn't look like the right file.")
             raise BulkImportException(msg, 'error')
 
-        csvcontent = StringIO(r.text)
+        encoding = r.encoding or 'utf-8'
+        content = r.text.encode(encoding).decode('utf-8')
```

Comment:
In principle, this could be done using requests itself (see [this](http://docs.python-requests.org/en/latest/user/quickstart/#response-content)):

```python
r.encoding = 'utf-8'
r.text # now it's encoded in utf-8 ;-)
```